File: NotificationSystem/NotificationSystem.Data/Classes/clsTypeList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace NotificationSystem.NotificationSystem.Data.Classes
{
	public class clsTypeList
	{
		
		private string m_Type = string.Empty;
		private int m_TroubleTicketNo = 0;

   
        public string Type
        {
            get
            {
                return m_Type;
            }
            set
            {
                m_Type = value;
            }
        }

        public int TroubleTicketNo
        {
            get
            {
                return m_TroubleTicketNo;
            }
            set
            {
                m_TroubleTicketNo = value;
            }
        }
    }
}

File: NotificationSystem/NotificationSystem.Data/Classes/clsTypeList.vb

Namespace NotificationSystem.NotificationSystem.Data.Classes
    Public Class clsTypeList

        Private m_Type As String = String.Empty
        Private m_TroubleTicketNo As Integer = 0


        Public Property Type As String
            Get
                Return m_Type
            End Get
            Set(value As String)
                m_Type = value
            End Set
        End Property

        Public Property TroubleTicketNo As Integer
            Get
                Return m_TroubleTicketNo
            End Get
            Set(value As Integer)
                m_TroubleTicketNo = value
            End Set
        End Property
    End Class
End Namespace
